### ISSUE TITLE
arcface python的tensorrt代码报错

### ISSUE THREAD

**Issue by hande6688** [wang-xinyu/tensorrtx#777]
## Env

作者的dockersfile安装
docker run --gpus all --name arcface_tensorrt -it -v $PWD:/app hakuyyf/tensorrtx:trt7_cuda10 /bin/bash

rm /etc/apt/sources.list.d/timsc-ubuntu-opencv-3_3-bionic.list
rm /etc/apt/sources.list.d/timsc-ubuntu-opencv-3_3-bionic.list.save
add-apt-repository -y ppa:timsc/opencv-3.4

apt-get update
apt-get install -y libopencv-dev libopencv-dnn-dev libopencv-shape3.4-dbg

pip install tensorrt-7.0.0.11-cp36-none-linux_x86_64.whl 
apt install python3-pycuda
apt-get install python-opencv
## About this repo

git clone https://github.com/deepinsight/insightface
cd insightface
git checkout 3866cd77a6896c934b51ed39e9651b791d78bb57
cd deploy
// copy tensorrtx/arcface/gen_wts.py to here(insightface/deploy)
// download model-r50-am-lfw.zip and unzip here(insightface/deploy)
pip install mxnet-cu101==1.7 -i https://pypi.tuna.tsinghua.edu.cn/simple
python gen_wts.py
gen_wts.py代码如下：
代码如下：
# -*- coding: utf-8 -*-
import torch
from torch import nn
import torchvision
import os
import struct
import argparse
from backbones import get_model
import cv2
import numpy as np
def main(args):
    net = get_model(args.network, dropout=0, fp16=False, num_features=args.ndim).cuda()
    weight = torch.load(args.prefix)
    net.load_state_dict(weight)
    device = torch.device(args.gpu)
    net.to(device)
    net.eval()
    # #生成wts
    if os.path.exists('arcface-r50.wts'):
        return
    f = open("arcface-r50.wts", 'w')
    f.write("{}\n".format(len(net.state_dict().keys())))
    for k,v in net.state_dict().items():
        print('key: ', k)
        print('value: ', v.shape)
        vr = v.reshape(-1).cpu().numpy()
        f.write("{} {}".format(k, len(vr)))
        for vv in vr:
            f.write(" ")
            f.write(struct.pack(">f", float(vv)).hex())
        f.write("\n")

if __name__ == '__main__':
    parser = argparse.ArgumentParser(description='face model test')
    # general
    parser.add_argument('--network', default='r50', help='Backbone network mobile0.25 or resnet50')
    parser.add_argument('--prefix', default='src_model/backbone.pth',help='path to load model.')
    parser.add_argument('--save_model', action="store_true", default=False, help='save full model')
    parser.add_argument('--gpu', default=0, type=int, help='gpu id')
    parser.add_argument('--ndim', default=512, type=int, help='embedding size')
    args = parser.parse_args()
    main(args)

运行上述代码没有问题
cd tensorrtx/arcface
// download joey0.ppm and joey1.ppm, and put here(tensorrtx/arcface)
mkdir build
cd build
cmake ..
make
执行 ./arcface-r50 -s  
报错：
Loading weights: ../arcface-r50.wts
[10/22/2021-17:20:16] [E] [TRT] Parameter check failed at: ../builder/Network.cpp::addScale::434, condition: shift.count > 0 ? (shift.values != nullptr) : (shift.values == nullptr)
arcface-r50: /app/tensorrtx-master/arcface/arcface-r50.cpp:107: nvinfer1::IScaleLayer* addBatchNorm2d(nvinfer1::INetworkDefinition*, std::map<std::__cxx11::basic_string<char>, nvinfer1::Weights>&, nvinfer1::ITensor&, std::__cxx11::string, float): Assertion `scale_1' failed.
Aborted (core dumped)

利用arcface-torch生成的model生成wts，只修改了wts的代码，其他的C++代码都采用作者的，
请问这是什么原因，有人知道吗？
